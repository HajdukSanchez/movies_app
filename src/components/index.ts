export { MovieList } from './movieList/MovieList';
export { MoviePoster } from './moviePoster/MoviePoster';
export { Loading } from './loading/Loading';
export { MovieDetails } from './movieDetails/MovieDetails';
export { ActorCard } from './actorCard/ActorCard';
