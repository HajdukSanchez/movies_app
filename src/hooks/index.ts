export { useMovies } from './useMovies/useMovies';
export { useMovieDetails } from './useMovieDetails/useMovieDetails';
