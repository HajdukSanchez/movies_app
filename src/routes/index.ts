/**
 * Params for Sack navigation.
 */
export type RootStackParamList = {
  Home: undefined;
  Detail: undefined;
};
