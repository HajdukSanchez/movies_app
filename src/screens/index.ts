export { DetailScreen } from './detailScreen/DetailScreen';
export { HomeScreen } from './homeScreen/HomeScreen';
