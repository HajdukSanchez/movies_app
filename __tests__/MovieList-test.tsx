import React from 'react';
import { render } from '@testing-library/react-native';

import { MovieList } from '../src/components';
import { Movie } from '../src/common/interface/movies.interface';

let movies: Movie[] = [];

describe('<MovieList/>', () => {
  test('should render correctly', () => {
    const list = render(<MovieList movies={movies} />);
    expect(list).toMatchSnapshot();
  });
});
