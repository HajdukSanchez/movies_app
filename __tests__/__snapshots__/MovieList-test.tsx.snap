// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieList/> should render correctly 1`] = `
<View
  style={
    Object {
      "paddingBottom": 10,
      "paddingTop": 5,
    }
  }
>
  <RCTScrollView
    ItemSeparatorComponent={[Function]}
    ListFooterComponent={[Function]}
    ListHeaderComponent={[Function]}
    data={Array []}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={true}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    showsHorizontalScrollIndicator={false}
    snapToInterval={2}
    stickyHeaderIndices={Array []}
    viewabilityConfigCallbackPairs={Array []}
  >
    <View>
      <View
        onLayout={[Function]}
      >
        <View
          style={
            Object {
              "marginHorizontal": 10,
            }
          }
        />
      </View>
      <View
        onLayout={[Function]}
      >
        <View
          style={
            Object {
              "marginHorizontal": 10,
            }
          }
        />
      </View>
    </View>
  </RCTScrollView>
</View>
`;
